client_plugin/Makefile

```diff
@@ -270,6 +276,7 @@ DEPLOY_VM_SH  := $(DEPLOY_TOOLS) deployvm
 DEPLOY_VM_TEST_SH  := $(DEPLOY_TOOLS) deployvmtest
 DEPLOY_VM_PLUGIN_SH := $(DEPLOY_TOOLS) deployplugin
 DEPLOY_ESX_SH := $(DEPLOY_TOOLS) deployesx
+DEPLOY_ESX_FOR_UPGRADE := $(DEPLOY_TOOLS) deployesxForUpgrade
```

Comment:
deployesxForUpgrade >> deployesxforupgrade to make it consistent with other targets